.github/workflows/autobuild.yaml

```diff
@@ -71,6 +71,10 @@ jobs:
             "-DLLVM_STATIC_LINK_CXX_STDLIB=ON"
             "-DLLVM_ENABLE_ZLIB=FORCE_ON"
             "-DCMAKE_PROJECT_INCLUDE=$GITHUB_WORKSPACE/.github/workflows/linux-static-deps.cmake"
+            "-DCLANGD_ENABLE_REMOTE=ON"
+          # This is needed to avoid dynamic linking of additional libraries.
+          grpc_cmake: >
+            "-DgRPC_CARES_PROVIDER=package"
```

Comment:
there are other dependencies like `absl`, `zlib`, `re2` and many more. afaict, they are all being build from source. are you saying that `c-ares` is the only one that will get dynamically linked even if it is build through `module` like other dependencies? If that's the case, it might be better to put that on the comment above (e.g. everything else can be built from source, but this one needs to use system installed package instead).

In addition to that, not sure if we care but in theory we can cut some build times by using system installed packages for others as well.

Reply:
Yeah that's the reason. Added the comment, thanks!

Building gRPC takes 7 minutes so it's not the bottleneck for sure.